build/prbuild.yml

```diff
@@ -52,7 +52,8 @@ jobs:
         **\*test*.dll
         !**\obj\**
       vsTestVersion: 15.0
-      codeCoverageEnabled: false
+      codeCoverageEnabled: true
+      publishRunAttachments: true
```

Comment:
>true [](start = 29, length = 4)

According to [https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/vstest?view=azure-devops](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/vstest?view=azure-devops), the default value for publishRunAttachments is already true, so I wouldn't expect this to have any impact

Reply:
the weird thing, it still don't publish it!